bst_plugin/process_nst_compute_glm.m

```diff
@@ -35,7 +35,7 @@
     % Description the process
     sProcess.Comment     = 'Compute GLM';
     sProcess.Category    = 'Custom';
-    sProcess.SubGroup    = '[wip]GLM';
+    sProcess.SubGroup    = 'wip - GLM';
```

Comment:
Let's use a more generic subgroup name "NIRS - wip". It will be easier for us and clearer for other users that everything in there is in-dev.
Also we will to handle wip processes during the installation and create a dedicated MANIFEST.wip